package-overrides/github/a5hik/ng-sortable@1.1.9.json

```diff
@@ -0,0 +1,17 @@
+{
+  "dependencies": {
+    "css": "*"
+  },
+  "directories": {
+    "lib": "dist"
+  },
+  "main": "ng-sortable",
+  "registry": "jspm",
+  "shim": {
+    "ng-sortable": {
+      "deps": [
+        "./ng-sortable.min.css!"
```

Comment:
This is to reproduce the css + js dependency here https://github.com/a5hik/ng-sortable/blob/1.1.9/bower.json#L8-L11